source/api_documentation/network_integration/js_tags/index.rst

```diff
@@ -0,0 +1,340 @@
+Invoca Tags
+============
+
+Manage Invoca Tags (aka JS Tags)
+""""""""""""""""""""""""""""""""""""""""""""""""
+
+The Invoca Tag is a snippet of code that connects Invoca to your landing pages. This API documentation helps provide necessary features to create and manage an Invoca Tag and it's revisions.
+
+.. raw:: html
+
+    <p>
+    More details on the Invoca Tags can be found <a href="https://community.invoca.com/t5/call-tracking/how-to-configure-and-deploy-your-invoca-tag-with-the-invoca-tag/ta-p/465#U465">here</a> and <a href="https://community.invoca.com/t5/developer-features/invoca-tag-wizard-custom-code-settings/ta-p/691">here</a>.
+    </p>
+
+.. list-table::
+  :widths: 15 15 50
+  :header-rows: 1
+  :class: parameters
+
+  * - Property
+    - Type
+    - Value
+
+  * - id
+    - integer (read-only)
+    - The internal Invoca id of the Invoca Tag.
+
+  * - name
+    - string
+    - The name of the Invoca Tag. Unique within network.
+
+  * - description
+    - string
+    - Invoca Tag description.
+
+  * - landing_page_url
+    - string
+    - Website or page URL on which the destination is applied.
+
+  * - active
+    - boolean
+    - Current status of the Invoca Tag.
+
+  * - status
+    - string
+    - Current status.
+
+  * - created_at
+    - string (read-only)
+    - Alias for the destination. Auto-generated names.
+
+  * - **live_revision**
+    - object
+    - Currently live revision.
+
+  * - **draft_revision**
+    - object
+    - Current draft revision.
+
+JS Tag Revision
+""""""""""""""""""""""""""""""""""""""""""""""""
+
+Revision for the JS Tag
+
+.. warning::
+    The provided documentation and JSON response for JS Tag Revision documentation might change in the next release.
+
+.. list-table::
+  :widths: 15 15 50
+  :header-rows: 1
+  :class: parameters
```

Comment:
Added the parameters here because we are returning revisions in all APIs. It might be more feasible to move these parameters to revisions documentation and add a link here.

Reply:
I think it's fine to leave the documentation here as well.  Keeps users from needing to tab to another endpoint's page despite only using this one.

Reply:
I wanted to avoid duplication. Let me see if I can create include it using a file.

Reply:
It is possible to include files. Moved it to a new file that's now included in JS tags documentation.